Source/WebKit/Shared/TextExtractionToStringConversion.cpp

```diff
@@ -395,7 +413,10 @@ static void addPartsForItem(const TextExtraction::Item& item, std::optional<Node
             aggregator.addResult(line, WTFMove(parts));
         },
         [&](const TextExtraction::TextFormControlData& controlData) {
-            parts.append(item.nodeName.convertToASCIILowercase());
+            if (aggregator.useTagNameForTextFormControls())
+                parts.append(item.nodeName.convertToASCIILowercase());
+            else
+                parts.append("textFormControl"_s);
```

Comment:
nit (and entirely opinion-based):
```suggestion
            const auto name = aggregator.useTagNameForTextFormControls() ? item.nodeName.convertToASCIILowercase() : "textFormControl"_s;
            parts.append(name);
```

Reply:
Sounds good! Condensed this down to one `append` call.